branchtesting/src/main/java/branchexample/App.java

```diff
@@ -4,13 +4,14 @@
 {
     public void aMethod( String[] args )
     {
-    	// TODO fix this
     	int i = 0;
-    	if (i == 0) {
+    	if (i != 0) {
     		// TODO fix this too..
     		int j = 5 / i;
     	}
+    	// TODO Add logger here
     	System.out.println("IN AMETHOD"); 
+    	//FIXME - fix this garbage
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Take the required action to fix the issue indicated by this comment. ([squid:S1134](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS1134))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=23&issues=AWbFpPH7LpqqSUlc1mdu&open=AWbFpPH7LpqqSUlc1mdu)

[ISSUE_KEY]: AWbFpPH7LpqqSUlc1mdu